src/features/bookOffer/pages/BookingConfirmation.native.test.tsx

```diff
@@ -47,69 +54,89 @@ describe('<BookingConfirmation />', () => {
     expect(page).toMatchSnapshot()
   })
 
-  it('should go to Bookings and log analytics event', async () => {
-    const renderAPI = render(<BookingConfirmation />)
-    fireEvent.press(renderAPI.getByText('Voir ma réservation'))
-    await waitForExpect(() => {
-      expect(analytics.logSeeMyBooking).toBeCalledWith(mockOfferId)
-      expect(reset).toBeCalledWith({
-        index: 1,
-        routes: [
-          {
-            name: 'TabNavigator',
-            state: {
-              routes: [{ name: 'Bookings' }],
-              index: 0,
-            },
-          },
-          {
-            name: 'BookingDetails',
-            params: {
-              id: 345,
-            },
-          },
-        ],
-      })
+  describe('InAppReview', () => {
+    it('should call InAppReview Modal after 3 seconds if isAvailable and rules are respected', () => {
+      isAvailable.mockImplementationOnce(() => true)
+      const requestInAppReview = jest
+        .spyOn(reactNativeInAppReview, 'RequestInAppReview')
+        .mockResolvedValue(() => true)
+
+      render(<BookingConfirmation />)
+      jest.advanceTimersByTime(3000)
+      expect(requestInAppReview).toHaveBeenCalledTimes(1)
     })
-  })
 
-  it('should call InAppReview Modal after 3 seconds if isAvailable and rules are respected', async () => {
-    const requestInAppReview = jest
-      .spyOn(reactNativeInAppReview, 'RequestInAppReview')
-      .mockResolvedValue(() => true)
+    it('should not call InAppReview Modal if isAvailable is false', () => {
+      isAvailable.mockImplementationOnce(() => false)
+      const requestInAppReview = jest.spyOn(reactNativeInAppReview, 'RequestInAppReview')
 
-    render(<BookingConfirmation />)
-    jest.advanceTimersByTime(3000)
-    // FIXME(Lucasbeneston): Investigate on whether its normal or not to have 3 calls here
-    expect(requestInAppReview).toHaveBeenCalledTimes(3)
-  })
+      render(<BookingConfirmation />)
+      jest.advanceTimersByTime(3000)
+      expect(requestInAppReview).toHaveBeenCalledTimes(0)
```

Comment:
C'est du détail mais j'ai l'impression que not.toHaveBeenCalled() est plus clair que toHaveBeenCalledTimes(0)